app/helpers/ui_constants.rb

```diff
@@ -474,7 +475,8 @@ module UiConstants
     "enterprise"            => _("The Enterprise"),
     "ext_management_system" => _("Selected %{tables}") % {:tables => ui_lookup(:tables => "ext_management_systems")},
     "ems_cluster"           => _("Selected %{tables}") % {:tables => ui_lookup(:tables => "ems_cluster")},
-    "vm-tags"               => _("Tagged %{tables}") % {:tables => ui_lookup(:tables => "vm")}
+    "vm-tags"               => _("Tagged %{tables}") % {:tables => ui_lookup(:tables => "vm")},
+    "tenant"		            => _("Tenant")
```

Comment:
Maybe it's tabs instead of spaces?  Either way, we can fix this after this gets merged.